spec/controllers/activities_controller_spec.rb

```diff
@@ -30,45 +36,44 @@
       end.to change { Activity.count }.by(1)
     end
 
+    context 'without lng and lat' do
+      let(:post_data) do
+        {
+          activities: [
+            {
+              location: 'outside',
+            }
+          ]
+        }
+      end
+
+      it 'demands lng and lat' do
+        SmarfDoc.skip
+        post :create, post_data
+        expect(
+          JSON.parse(response.body)
```

Comment:
I like having a helper for parsing the json responses, but this works, especially  if you're just trying to get it out the door.